### PR TITLE
Hotfix/agreements funds

### PR DESCRIPTION
**Pull request by farinalvaro**
- calculates withdrawable funds for consumer
- calculates funds to be paid out for provider
- fixes listedSize amount to show totalCapacity in offers
- removes TimePeriodEnum (again)

### CODE REVIEW

**Review by julianmrodri — APPROVED** [review]
LGTM Magic in the air!

File: src/api/rif-marketplace-cache/storage/__tests__/offers.test.ts [hunk 1]
```diff
@@ -89,6 +90,7 @@ describe('Storage OffersService', () => {
         averagePrice: avgBillingPrice,
         peerId: 'QmYyQSo1c1Ym7orWxLYvCrM2EmxFTANf8wXmmE7DWjhx5N',
         utilizedCapacityGB: new Big(utilizedCapacity).div(UNIT_PREFIX_POW2.KILO),
+        totalCapacityGB: new Big(totalCapacity).div(UNIT_PREFIX_POW2.KILO),
```
Comment (jurajpiar) [inline src/api/rif-marketplace-cache/storage/__tests__/offers.test.ts:93]: Might be worth to use this opportunity to extract a conversion function?

File: src/api/rif-marketplace-cache/storage/offers.ts [hunk 2]
```diff
@@ -54,6 +55,7 @@ const mapFromTransport = (offerTransport: OfferTransport): StorageOffer => {
     acceptedCurrencies,
     peerId,
     utilizedCapacityGB: new Big(utilizedCapacityMB).div(UNIT_PREFIX_POW2.KILO),
+    totalCapacityGB: new Big(totalCapacityMB).div(UNIT_PREFIX_POW2.KILO),
```
Comment (jurajpiar) [inline src/api/rif-marketplace-cache/storage/offers.ts:58]: same here

**Review by jurajpiar — APPROVED** [review]
Good job

**Review by jurajpiar — APPROVED** [review]

**Review by julianmrodri — APPROVED** [review]
LGTM

**Pull request merged**